### ISSUE TITLE
As a content editor requiring an accessible content admin area, I want to be able to use the main menu screen without WCAG AA compliance issues so that I can effectively manage the site

### ISSUE THREAD

**Issue by kurtfoster** [govCMS/GovCMS7#239]
`/admin/structure/menu/manage/main-menu`
### Errors

Empty link
### What It Means

A link contains no text.
### Why It Matters

If a link contains no text, the function or purpose of the link will not be presented to the user. This can introduce confusion for keyboard and screen reader users.
### How to Fix It

Remove the empty link or provide text within the link that describes the functionality and/or target of that link.
### The Algorithm... in English

An anchor element has an href attribute, but contains no text (or only spaces) and no images with alternative text.
### Standards and Guidelines

2.4.4 Link Purpose (In Context) (Level A)
Icon index

**Comment by Chris820** [comment]
Would this be about those drag to re-order arrows then?

Thys seem to be used in a bunch of pages within Drupal's admin.

I suppose those links do have title attributes. And even if the support for reading out title attribs within user agents is pretty terrible - it is _technically_ a sufficient technique (H33 for WCAG2 2.4.4). 
(IMO, Title attribs alone are insufficient.)

But regardless: There is still that 'show row weights' link option. Which turns off the draggable re-order method and replaces it with explicity labelled select drop-downs. Also needed for WCAG2 2.1.1

**Comment by kurtfoster** [comment]
@Chris820, yes, this relates to the reorder arrows. Sorry for the lack of context, this was a quick entry based on a client's testing to allow me to clear out a backlog item. It's not a real show stopper, but given this is for Govt, it probably should get a patch to make it compliant.

**Comment by fiasco** [comment]
We're happy to support WCAG compliance but this particular issue is not on our radar to address directly. But we would happily review any pull request to address these types of issues.
